### DISCUSSION TITLE
multiple database connection define same models?

### DISCUSSION THREAD

**Discussion by erhuabushuo** [emmett-framework/emmett#396] [General]
multiple database connection define same models?

**Comment by gi0baro** [comment]
@erhuabushuo it's not supported, as you can call the model class directly (eg: `MyModel.first()`).

You can, thus, define your common model as a class, and then inherit, eg:

```python
class Todo(Model):
    ...


class TodoA(Todo):
    tablename = "todos"


class TodoB(Todo):
    tablename = "todos"


db1.define_models(TodoA)
db2.define_models(TodoB)
```
